src/storage.js

```diff
@@ -0,0 +1,21 @@
+export const storageFactory = {
```

Comment:
I don't think it is a factory. It seems a regular implementation of the storage. If so, we should rename it to `LocalStorageStore`.

Reply:
We should also rename the file to `src/store.js`.

Reply:
Done. Renamed to `MobileStore`.

Reply:
Renamed and transformed into a class, but kept the name `storage.js` so it is not confused with what's inside the folder `store`, which is related to redux.

Reply:
Ok.